src/Testing/src/AssemblyTestLog.cs

```diff
@@ -218,12 +223,20 @@ public static AssemblyTestLog ForAssembly(Assembly assembly)
         {
             if (!_logs.TryGetValue(assembly, out var log))
             {
-                var baseDirectory = TestFileOutputContext.GetOutputDirectory(assembly);
+                var stackTrace = Environment.StackTrace;
+                if (!stackTrace.Contains(
+                    "Microsoft.AspNetCore.Testing"
```

Comment:
Is this ever false? Wouldn't it include the current method which is in Microsoft.AspNetCore.Testing.

Reply:
Thought it might be but you're probably right. I'll play with this in a follow-up when I get a bit of time.